app/src/main/java/net/justdave/mcstatus/MinecraftServer.java

```diff
@@ -235,9 +231,11 @@ public String description() {
 	}
 
 	public Bitmap image() {
-		String imagedata = serverJSON.optString("favicon");
+		String imageData = serverJSON.optString("favicon");
+		if (imageData.isEmpty())
```

Comment:
This prevents problems in the `getThumbnail` function when the URI is empty. The exception was caught, but not throwing is even nicer :)

Reply:
Good idea.